corehq/apps/app_manager/templates/app_manager/v1/partials/form_tab_advanced.html

```diff
@@ -2,6 +2,7 @@
 {% load xforms_extras %}
 {% load hq_shared_tags %}
 <div class="tab-pane" id="advanced">
+    {% if form.form_type != "shadow_form" %}
```

Comment:
This should be moved to hide the entire advanced tab for shadow forms (since it looks like `allow_form_copy` is only advanced forms, which will make this tab always blank for shadow forms).